sycl/test-e2e/README.md

```diff
@@ -67,6 +67,19 @@ CC=<> CXX=<> python llvm/buildbot/configure.py -o build ... \
   --cmake-opt=-DSYCL_E2E_TESTS_LIT_FLAGS="--param;dump_ir=True"``
 ```
 
+In an in-tree build, individual tests or groups of tests can be conveniently
+run directly from their source paths, using the configured `llvm-lit` script:
+
+```
+# Implicitly uses cmake parameters SYCL_BE and SYCL_TARGET_DEVICES, detailed
+# below
```

Comment:
It seems it requires specifying them manually, in addition to `SYCL_TEST_E2E_TARGETS` above. Maybe worth adding a note somewhere here?